WalletWasabi.Fluent/Views/AddWallet/WelcomePageView.axaml

```diff
@@ -243,7 +243,7 @@
           </Viewbox>
           <DockPanel Grid.Column="0" Grid.Row="0" VerticalAlignment="Center">
             <TextBlock Classes="text"
-                       Text="Wasabi provides privacy by default, it protects your financial information through coinjoins, client-side block filtering and communication over the Tor anonymity network.&#10;&#10;Neither the public nor the developers can breach your privacy." />
+                       Text="Wasabi provides privacy by default.&#10;&#10;Nobody can find out how much money you have, or where you spend it.&#10;&#10;For that you pay a fee of 0.3%." />
```

Comment:
Maybe we should have a dedicated dialog to explain just the fees, highlighting mining/coordinator fees and the discounts.

Reply:
```suggestion
                       Text="Wasabi provides privacy by default.&#10;&#10;Nobody can find out how much money you have, or where you spend it." />
```